deployment/deploy.sh

```diff
@@ -18,35 +18,67 @@ source $CURRENT_DIR/vars/$DEPLOY_ENV.sh
 yarn global add serverless@1.32.0
 yarn
 
+# Create s3 bucket
+node $CURRENT_DIR/createS3Bucket.js
+if [ $? != 0 ]; then
+  echo "Bucket build failed"
+  exit 1
+fi
+
 # Deploy with serverless
 sls deploy -v | tee out.tmp
 if [ "${PIPESTATUS[0]}" != "0" ]; then
   echo "sls deploy failed"
   exit 1
 fi
 
+# Check if seeding will be required
+node $CURRENT_DIR/../db/scripts/writeSeedFlag.js
+if [ $? != 0 ]; then
+  echo "seed flag script failed"
+  exit 1
+fi
+
+# Initialize floods database
+node $CURRENT_DIR/../db/scripts/initialize.js
```

Comment:
I'm assuming the "do we need to init?" logic is in initialize.js

Reply:
Yep. It checks that floods exists first.